charts/harvester-cloud-provider/102.0.2+up0.2.3/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Harvester Cloud Provider
+  catalog.cattle.io/kube-version: '>= 1.23.0-0 < 1.27.0-0'
```

Comment:
@w13915984028 Rancher team is going to support v1.27 in 2.7, so we need to bump the condition here.
See https://github.com/rancher/rancher/issues/44126 for the new issue.